tests/testFiles.json

```diff
@@ -4,16 +4,20 @@
   "spec/core/biblio-db-spec.js",
   "spec/core/biblio-spec.js",
   "spec/core/caniuse-spec.js",
+  "spec/core/data-abbr-spec.js",
```

Comment:
Oops. Didn't know this.

Reply:
It's ok, it's only for local testing :) I keep forgetting to update this file too.... hence all the changes below.